### DISCUSSION TITLE
RAG not able to summarize uploaded file

### DISCUSSION THREAD

**Discussion by tkg61** [open-webui/open-webui#5306] [General]
# Bug Report

## Installation Method

Helm install of 0.3.21

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.
- [x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

When i ask for a summary of a file, it gives me a summary

## Actual Behavior:

It says it doesn't know what i am talking about 

## Description

**Bug Summary:**

the system is not recognizing that a file is uploaded and says it doesn't know what to do

## Reproduction Details

**Steps to Reproduce:**
Run Openwebui like normal
upload any file
ask it to summarize the file

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots/Screen Recordings (if applicable):**

![image](https://github.com/user-attachments/assets/11f7514e-929d-478b-89c5-c8d2dcff15f7)

## Additional Information

When i ask it a specific question it can answer it but the summarize query should work: 

![image](https://github.com/user-attachments/assets/237b25f6-3077-49fe-a0d1-2dc9e6a2bd49)

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by peuportier** [comment]
HI @tkg61  What model are you using for embedding ? can you provide us more detail about the configuration you are using concerning the embedding ? Did you have install tika , what your chunk parameter ... Please go on admin settings and Document to let us know. Thanks you

**Comment by thiswillbeyourgithub** [comment]
Pretty sure it has something to do with https://github.com/open-webui/open-webui/discussions/5202#discussioncomment-10578585

**Comment by tkg61** [comment]
I am using all the defaults in Openwebui (model, content extraction, rag template, etc). All of it is unmodified.

**Comment by tkg61** [comment]
I did a little more experimentation. It seems like keeping the defaults but just adding in Tika did not solve anything.

I then experimented with a different embedding model (BAAI/bge-small-en-v1.5) and that seemed to have helped since i was able to do the same prompt and have it summarize what i was talking about. 

The stranage thing is that if i reset everything (upload dir, vector db) and switch all my settings back to normal out of the box defaults. i can sometimes get the same prompt to work. 

this just seems so random without having insight into what is going on. It would be cool like @thiswillbeyourgithub said to have more insight into whats happening since from the gui right now it looks like the file should have been used
